spec/requests/users_controller_spec.rb

```diff
@@ -4651,8 +4660,8 @@ def create_totp
       end
       context 'when user has a registered totp and security key' do
         before do
-          totp_second_factor = Fabricate(:user_second_factor_totp, user: user1)
-          security_key_second_factor = Fabricate(:user_security_key, user: user1, factor_type: UserSecurityKey.factor_types[:second_factor])
+          _totp_second_factor = Fabricate(:user_second_factor_totp, user: user1)
```

Comment:
What's with these changes to add `_` ? I know they are for unused variables, is this a rubocop auto fix thing? We should just delete the variable IMO

Reply:
Bearing in mind, I probably added these myself :laughing: `security_key_second_factor`

Reply:
done mechanically cause ale was complaining about Ruby warnings (unused vars) , we should probably just lint for this.

Reply:
@SamSaffron weird that `ale` is not using our rubocop rules, this is not one of our rules afaict